quotes_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"log"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestLoadQuotes(t *testing.T) {
+	log.Println(quoteList)
+
+	// Define quotes
+	quotes := map[int]string{
+		0: "Be yourself; everyone else is already taken. - Oscar Wilde",
+		1: "In the end, it's not the years in your life that count. It's the life in your years. - Abraham Lincoln",
+		2: "Who is a greater reprobate than he, Who feels compassion at the doom divine? - Virgil",
+		3: "Opinion is the medium between knowledge and ignorance. - Plato",
+	}
+
+	// Build quotes into quotefile format
+	var quotestr string
+	for i := 0; i < len(quotes); i++ {
+		quotestr += quotes[i]
+		if i+1 != len(quotes) {
+			quotestr += "\n"
+		}
+	}
+
+	// Parse quotefile
+	r := strings.NewReader(quotestr)
+	loadQuotes(r)
+
+	// Compare parsed values against originals
+	for key, val := range quotes {
+		if quoteList[key] != val {
+			t.Errorf(`error, expected "%s", got "%s"`, val, quoteList[key])
+		}
+	}
```

Comment:
This test is a welcomed addition. My bigger concern is #7 with a few  invariants I worry about.

- The "default" search always returns all quotes
- Searches return the expected quotes
- New quotes get added to the end of pre-existing searches
- There was a bug in a previous version that once a search ran until completion it would start returning `""` instead of restarting. Exhausting a quote should just restart the search (with or without a new random order)